docs/reference/changelog.rst

```diff
@@ -9,6 +9,7 @@ X.Y.Z (2025-MM-DD)
 Bug fixes:
 
 - Correctly handle ``source-subdir`` values on the ``go-use`` plugin.
+- Preserve the ``pcfiledir`` tag in ``pkgconfig`` files.
```

Comment:
We've been unusually busy in craft-parts this month and just released yesterday :upside_down_face:.  Can you add a new entry?

```
X.Y.Z (2025-MM-DD)
------------------

Bug fixes:

- Preserve the ``pcfiledir`` tag in ``pkgconfig`` files.
```

I'll land this ASAP so we don't flap on this again.

Reply:
Done.